docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:9.6.5
+    volumes:
+    - postgres_data:/var/lib/postgresql/data/
+  redis:
+    image: "redis:alpine"
+  web:
+    build: .
+    command: bash -c "pipenv run python /src/manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - .:/code/src
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+      - redis
+  celery:
+    build: .
+    command: celery -A proj worker -l info
```

This should probably be:

```suggestion
    command: pipenv run celery -A proj worker -l info
```

---

is pipenv necessary since we're inside a docker image? Does it provide value over workspace specific dependencies?

---

I think we can't easily avoid this, since we don't have an old-fashioned `requirements.txt` file (which would be required for a global `pip` install). Using pipenv also ensures we're using the exact same dependencies, since it also provides a lockfile.